### DISCUSSION TITLE
How to pass extra data to Jetstream livewire component directly?

### DISCUSSION THREAD

**Discussion by lucyacat** [laravel/framework#34744] [Q&A]
I want to pass extra data to navigation-dropdown component directly from blade, so the menu is different for non logged in users, but it didn't work. I can make it work by adding a property to NavigationDropdown directly, but I don't want to change codes in the vendor directory.

Many thanks.

![image](https://user-images.githubusercontent.com/65777669/95438615-1a385100-0989-11eb-8bfa-d141538ab2ed.png)

**Comment by skybitbbsr** [comment]
+1

also, need to know how to pass in html way

`<livewire:navigation-dropdown extradata="true" />`

like this

**Comment by dennykurniawan48** [comment]
Hello, did you find the answer?

**Comment by wizzymore** [comment]
Create your own component that extends the jetstream one to add more functionality on top of it, otherwise there is nothing you can do other then copy pasting the component and moving it in your code.
